src/shared/output-mode.c

```diff
@@ -14,7 +14,10 @@ JsonFormatFlags output_mode_to_json_format_flags(OutputMode m) {
                 return JSON_FORMAT_SEQ;
 
         case OUTPUT_JSON_PRETTY:
-                return JSON_FORMAT_PRETTY;
+                return JSON_FORMAT_PRETTY | JSON_FORMAT_COLOR_AUTO;
+
+        case OUTPUT_JSON:
+                return JSON_FORMAT_PRETTY | JSON_FORMAT_PRETTY_AUTO;
```

Comment:
I cannot follow the change. Please drop this.

Reply:
For this, this is what sets the format variable to the desired format. when the option is passed, eg --output=json etc, there is a reverse lookup table and a functions that sets OUTPUT_JSON or the likes based on the option argument then this function here returns the JsonFormatFlag option based on this OUTPUT mode. This is what sets the value of the Json format flags variable. It does not work any longer when i drop it.